README.md

```diff
@@ -177,7 +177,7 @@ If the database does not exist and you have permissions, it will be created for
 * **config.database** : String (required) - database name, must be between 3 and 255 characters and must consist only of lower case letters, numbers, and _
 * **config.writeKey** : String (required) - write-only key, get it from your [user profile](console.treasuredata.com/users/current)
 * **config.protocol** : String (optional) - protocol to use for sending events. Default: result of `document.location.protocol`
-* **config.pathname** : String (optional) - path to append after host. Default: `/js/v3/event/`
```

Comment:
Uhm.... Correct yes (because up to now all paths used `/js/v3/event` but now we need to use `/js/v3/global_id`), best approach probably too. Backwards compatible no. We can make a non breaking change for this, by calculating the result endpoint for global_id from `hostname` instead of from `pathname`. Thoughts?